tests/FSharp.Test.Utilities/XunitHelpers.fs

```diff
@@ -139,9 +146,24 @@ type FSharpXunitFramework(sink: IMessageSink) =
         log "FSharpXunitFramework with XUNIT_EXTRAS installing TestConsole redirection"
         TestConsole.install()
 
+// TODO: Currently does not work with Desktop .NET Framework. Upcoming OpenTelemetry 1.11.0 may change it.
```

Comment:
out of curiosity, what makes you think 1.11 will enable .NET Framework support for OTel? I don't see anything in the [release notes](https://github.com/open-telemetry/opentelemetry-dotnet/blob/core-1.11.0-rc.1/RELEASENOTES.md) or [changelog](https://github.com/open-telemetry/opentelemetry-dotnet/blob/core-1.11.0-rc.1/src/OpenTelemetry/CHANGELOG.md) to suggest that.

Reply:
Is it this [new builder pattern](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5325)?

Reply:
They say they're dropping the quasi-deprecated gRpc.Core. When I tried it here, grpc just fails silently (exception can be observed in debug mode) during runtime on desktop target.

Reply:
May be wishful thinking on my part, though!

Reply:
I just did a quick test with console app. Otel 1.9.0 does work in general with net472, just not here in this repo, for some reason.